### DISCUSSION TITLE
Video Speed Manager

### DISCUSSION THREAD

**Discussion by IdleEndeavor** [minbrowser/min#1468] [Ideas]
I know not everyone would want this feature, but I fealt like it is a simple thing to add and would make the experience for productivity nerds like me a little better. I have the habit of watching videos at 3x speed. (I know I'm weird) and one of the things preventing me from switching over completely to Min Browser is indeed that.
If the creator is reading, please do consider adding a feature to control playback speed of videos globally across websites.
Thanks a Lot!

**Comment by Milo123459** [comment]
Not sure, but wouldn't we have to edit sites HTML?

**Comment by PalmerAL** [comment]
There’s a property you can set on the video element to make it work.

Probably a good candidate for a userscript.

**Reply by Milo123459** [reply]
Userscripy 100%
